applications/tari_validator_node/src/grpc/validator_node_grpc_server.rs

```diff
@@ -129,6 +154,7 @@ impl<TServiceSpecification: ServiceSpecification + 'static> rpc::validator_node_
         &self,
         request: Request<rpc::InvokeReadMethodRequest>,
     ) -> Result<Response<rpc::InvokeReadMethodResponse>, Status> {
+        println!("invoke_read_method grpc call");
```

Comment:
It's better to use `info!` etc, at some point all of these `println`'s should be replaced